src/txn/txn_rollback_to_stable.c

```diff
@@ -93,6 +108,32 @@ __rollback_abort_newer_insert(
               session, ins->upd, rollback_timestamp, &stable_update_found);
 }
 
+/*
+ * __rollback_col_add_update --
+ *     Add the provided update to the head of the update list.
+ */
+static inline int
+__rollback_col_add_update(WT_SESSION_IMPL *session, WT_REF *ref, WT_UPDATE *upd, uint64_t recno)
```

Comment:
Why not call it `__rollback_col_modify` as we already have `__wt_hs_modify`?

Reply:
Yeah, that makes sense.